lib/generators/webpack_rails_react/install_generator.rb

```diff
@@ -70,23 +70,48 @@ def create_webpack_application_js
         copy_file "boilerplate/App.js", "webpack/containers/App.js"
       end
 
-      application_view = 'app/views/layouts/application.html.erb'
+      haml_installed = Gem.loaded_specs.has_key? 'haml-rails'
+      layouts_dir = 'app/views/layouts'
 
-      if File.exists? application_view
-        insert_into_file 'app/views/layouts/application.html.erb', before: /<\/body>/ do
-            <<-'RUBY'
-<%= ja  vascript_include_tag *webpack_asset_paths('application') %>
-            RUBY
+      application_view = haml_installed ? "#{layouts_dir}/application.html.haml" : "#{layouts_dir}/application.html.erb"
+
+      if haml_installed
+        # convert views to haml
+        begin
+          require 'html2haml'
+        rescue LoadError
+          `gem install html2haml`
+        end
```

Comment:
Is this actually converting the users erb file to haml if they have the gem installed?  If so I would rather leave it as whatever it currently is and insert the proper tags.